docs/sources/flow/reference/components/loki.process.md

```diff
@@ -330,6 +330,30 @@ stage.labels {
 }
 ```
 
+### stage.non_indexed_labels block
+
+The `stage.non_indexed_labels` inner block configures a labels processing stage that can read
+data from the extracted values map and add them to log entries as non-indexed labels.
+
```

Comment:
Could you explain here what the non-indexed labels and refer to Loki's docs for further reading?

Reply:
Yes, but Loki docs are not published yet. once it's published I will add the link to the page